x-pack/plugins/security_solution/common/endpoint/data_generators/fleet_action_generator.ts

```diff
@@ -11,21 +11,62 @@ import { BaseDataGenerator } from './base_data_generator';
 import { EndpointAction, EndpointActionResponse, ISOLATION_ACTIONS } from '../types';
 
 const ISOLATION_COMMANDS: ISOLATION_ACTIONS[] = ['isolate', 'unisolate'];
+interface OSQueryData {
```

Comment:
we shouldn't know anything about this

Reply:
@pzl should I rename these types then? Like `NonEndpointData` perhaps 🤔

Reply:
That could be better.

And I wouldn't necessarily rely on the exact structure of what osquery does. Any other plugin could be writing any shape of actions into that index at some point in the future (even mid-release, if people install or write their own plugins). So it's less important that we know about anyone's else's document shape, and more that we are able to filter out *every* other arbitrary data or shape that is not our own. So what that other data is should be irrelevant. 

I don't know that we want out _endpoint data generator_ to be creating fake-other-actions at all. It was likely useful for development here, but this is probably not where it belongs long-term

Reply:
I see what you mean @pzl. I'll remove these methods before this is ready for merge then.

Reply:
done c5227c092a3cc4d1fe20ae8d1de26cc0bc0194ff